### ISSUE TITLE
Chat view: show accessibility help does not work when focus is on request item

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#189135]
Refs: https://github.com/microsoft/vscode/issues/186619

1. Open chat view. Ask some things
2. Move keyboard focus on the request item (what you typed)
3. Press `Alt + F2` -> nothing happens 🐛 

This is inconsistent with the behaviour when the focus is on the response.

Megan informed me that this behaviour is by design, but I do not see understand what is the benefit of this behaviour. What are we getting with this inconsistency?

**Comment by meganrogge** [comment]
cc @jooyoungseo and @rperez030 for feedback.

The thought here was a user has typed the request - thus knows what it says generally and doesn't need to carefully inspect it. Also, the request is typically concise.

Open to adding support for this though - we'd have to consider if we want the `next` and `previous` to cycle through each list item. IMO, it would be a bit annoying to have to go through one's requests like this.

**Comment by meganrogge** [comment]
I discussed this with JooYoung and he agreed we should not do this unless it's requested
